### ISSUE TITLE
ul.navbar-nav being empty on load violates WCAG 1.3.1

### ISSUE THREAD

**Issue by cjg89** [UCF/Section-Menus-Shortcode#35]
**Describe the bug**

https://github.com/UCF/Section-Menus-Shortcode/blob/master/includes/sections-menu-common.php#L165-L167

This `<ul>` having no children on page load throws warnings in accessibility scans, even when it gets populated with list items via javascript.

**Steps to Reproduce**

1. Create a page with a section nav
2. Scan with Lighthouse, Axe dev tools, etc

**Expected Behavior**

Our generated markup should not violate accessibility requirements.

**Possible Fix**

Remove the `<ul>` from the navbar if there are not any `$list_items` already available to insert, and insert the `<ul>` via javascript instead when needed.

Additionally, we should consider removing the `nav.sections-menu-wrapper` entirely if there are no pre-populated nav items or nav items to insert via javascript.

**Your environment:**

n/a

**Additional context**

n/a

**Comment by cjg89** [comment]
Going to close this issue as we're no longer getting flagged in scans for this (SiteImprove's js crawler is probably able to properly detect js-populated nav links now.)
